clustermesh-apiserver/users_mgmt.go

```diff
@@ -96,7 +101,7 @@ func (us *usersManager) Start(hive.HookContext) error {
 		return fmt.Errorf("unable to setup config watcher: %w", err)
 	}
 
-	us.manager.UpdateController(usersMgmtCtrl, controller.ControllerParams{
+	us.manager.UpdateController(usersManagementControllerGroup, usersMgmtCtrl, controller.ControllerParams{
```

Comment:
This controller is run as part of the clustermesh-apiserver. To expose new metrics, you have to enable them here: https://github.com/cilium/cilium/blob/main/clustermesh-apiserver/metrics/metrics.go#L75-L87. The same goes for kvstoremesh here: https://github.com/cilium/cilium/blob/main/kvstoremesh/metrics/metrics.go#L89-L101.